app/src/androidTest/java/org/mozilla/fenix/ui/robots/HomeScreenRobot.kt

```diff
@@ -245,9 +245,15 @@ class HomeScreenRobot {
 
     fun verifyExistingTopSitesList() = assertExistingTopSitesList()
     fun verifyNotExistingTopSitesList(title: String) = assertNotExistingTopSitesList(title)
+    fun verifySponsoredShortcutDoesNotExist(sponsoredShortcutTitle: String, position: Int) =
+        assertSponsoredShortcutWithTitleDoesNotExist(sponsoredShortcutTitle, position)
```

Comment:
Let's avoid this format going forward, move the implementation directly into verifySponsoredShortcutDoesNotExist

Reply:
Done !